src/steps/initializeGitHubRepository/outcomeLabels.ts

```diff
@@ -68,10 +70,11 @@ export const outcomeLabels = [
 	},
 	{
 		color: "d73a4a",
-		description: "Something isn't working :( 🐛",
+		description: "Something isn't working 🐛",
```

Comment:
Technically this is a behavior change in the repo. Heh. I'm curious if anybody will notice. The combo of an emoticon and an emoji bugs me.